samples/android/GleanCrashExample/README.md

```diff
@@ -0,0 +1,13 @@
+# Glean Crash Example Application
+
+A simple app showcasing the Glean SDK used to instrument crashes in Android applications.
```

Comment:
I feel like saying that "lib-crash is the recommended way" totally invalidates this entire example.  Why even follow this then?  I used lib-crash as the basis for building this example.  Lib-crash is a "Crash Reporting Service" and not purely a crash telemetry instrumentation and usually is set up to require a user to confirm sending the metrics via a dialog.

Reply:
Please note that I mentioned `Mozilla products`. Our products are recommended/required to use Android Components rather than write their own custom things.

Reply:
This doesn't bother me as much after our conversation on Slack.  Especially since I'm removing the example, I can point to lib-crash as an example of a specific implementation.